odoo/models.py

```diff
@@ -750,6 +750,16 @@ def _init_constraints_onchanges(cls):
         cls._ondelete_methods = BaseModel._ondelete_methods
         cls._onchange_methods = BaseModel._onchange_methods
 
+    def _neutralize(self):
+        """ Neutralize this model's records.
+        This should prevent the database from connecting to external services.
+        """
+        return
+
+    def _neutralize_warning(self, message):
+        """ Neutralize warning helper """
+        _logger.warning("neutralize: %s", message)
```

Comment:
Is this really useful? It's going to emit all warnings from `odoo.models`, potentially making finding the issue much more difficult.

In pretty much all places where it's used, the method calling it already has a logger available.

Reply:
The purpose is the make is overridable, i.e. to crash instead of emitting a warning